framework/Distributions.py

```diff
@@ -439,6 +439,15 @@ def pdf(self,x):
     returnPdf = self._distribution.pdf(x)
     return returnPdf
 
+  def logpdf(self,x):
+    """
+      Function to get the log pdf at a provided coordinate
+      @ In, x, float, value to get the pdf at
+      @ Out, logpdf, float, requested log pdf
+    """
+    logpdf = np.log(self._distribution.pdf(x))
```

Comment:
shouldn't be just :
```logPdf = np.log(self.Pdf(x))?```

In this way we would not need to change this method if we decided to change the variable "self._distributiomn"